Application/EdFi.Ods.Features/ChangeQueries/ExceptionHandling/SnapshotGoneExceptionTranslator.cs

```diff
@@ -22,14 +23,14 @@ namespace EdFi.Ods.Features.ChangeQueries.ExceptionHandling
     /// </summary>
     public class SnapshotGoneExceptionTranslator : IExceptionTranslator
     {
-        private readonly ISnapshotContextProvider _snapshotContextProvider;
+        private readonly IContextProvider<SnapshotContext.SnapshotUsage> _snapshotContextProvider;
 
         /// <summary>
         /// Initializes a new instance of the <see cref="SnapshotGoneExceptionTranslator" /> class using the
         /// supplied snapshot context provider.
         /// </summary>
         /// <param name="snapshotContextProvider">Provides access to the snapshot context for the current request.</param>
-        public SnapshotGoneExceptionTranslator(ISnapshotContextProvider snapshotContextProvider)
+        public SnapshotGoneExceptionTranslator(IContextProvider<SnapshotContext.SnapshotUsage> snapshotContextProvider)
```

Comment:
Reduce this to `SnapshotUsage` by including the namespace with a `using` statement above.